train.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import re
+import yaml
+import os
+import hydra
+from hydra.core.config_store import ConfigStore
+from pathlib import Path
+
+
+# Hack to fix multiprocessing deadlock when PyTorch's DataLoader is used
+# (more info: https://github.com/pytorch/pytorch/issues/1355)
+import cv2
+cv2.setNumThreads(0)
+
+# from src.constructor import TASKS
+from src.constructor.config_structure import ConfigParams
+# from src.constructor import create_trainer
+
+
+@hydra.main()
+def load_config(config):
+    # Need to add --config-path (-cp) and --config_name (-cn) in run command
+    # Example config in configs/classification_cifar10.yaml
+    # Then the command line will be:
+    # python --config-path configs --config_name classification_cifar10 
+    config_params = ConfigParams(**config)
+    return config_params
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-cp', '--config-path', type=str,
```

Comment:
I think we should support just `config` parameter directing to the full config. It will be the only required argument

Reply:
Hydra not support only name it need path to

Reply:
But you aren't using `hydra.main`. Ok, let's move to the correct usage of hydra